pallets/settlement/src/lib.rs

```diff
@@ -565,14 +565,13 @@ decl_module! {
             settlement_type: SettlementType<T::BlockNumber>,
             valid_from: Option<T::Moment>,
             legs: Vec<Leg<T::Balance>>,
-            portfolios: Vec<PortfolioId>
+            portfolios: BTreeSet<PortfolioId>
```

Comment:
A merge went wrong? BTreeSet -> Vec changes should not have been reverted.

Reply:
Yeah I don't know what happened there. I'll revert it,